test/browser.js

```diff
@@ -17,14 +17,11 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
-require('tap-browser-color')();
```

Comment:
Can this be wrapped in a `try-catch` or `if (typeof document !== undefined` so that it still works when not running in headless mode?

Reply:
- In headless mode `document` is still present
- This breaks in both headless and non-headless. When a screenshot is being taken, `document` becomes temporarily unavailable.

The lib does not have that much code, we might as well re-implement it using the `logToDom` functions from probe.

Reply:
Sounds good, one less dependency.